HISTORY.rst

```diff
@@ -10,6 +10,7 @@ v0.26 (unreleased)
 * fix return type hint for ``create_model``, #526 by @dmontagu
 * **Breaking Change:** fix ``.dict(skip_keys=True)`` skipping values set via alias (this involves changing
   ``validate_model()`` to always returns ``Tuple[Dict[str, Any], Set[str], Optional[ValidationError]]``), #517 by @sommd
+* fix to schema generation for ``IPv4Address``, ``IPv6Address``, ``IPv4Interface``, ``IPv6Interface``, ``IPv4Network``, ``IPv6Network`` by @euri10
```

Comment:
can you add the PR number here like other changes.

Reply:
done